prov/usnic/src/usdf_cm.c

```diff
@@ -235,13 +238,32 @@ usdf_cm_msg_connreq_failed(struct usdf_connreq *crp, int error)
 		eq = pep->pep_eq;
 	}
 
+	reqp = (struct usdf_connreq_msg *) crp->cr_data;
```

Comment:
> does that mean you're going to rework the patch? Or should I review as-is?

Well I just realized that there are several places that this failure function is used, so if any of those have data associated with the request then the data will be added to the error entry. Not sure if that is what we want or not, I think the reject event might be a special case. I mean I don't think there is any harm in having that data around for non-reject events, just not sure if it should be there.